pycheribuild/projects/cross/cheribsd.py

```diff
@@ -355,26 +355,22 @@ def _stdout_filter(self, line: bytes):
 
     @property
     def arch_build_flags(self):
+        result = {
+            "TARGET_ARCH": self.target_info.target_arch,
+            }
         if self.compiling_for_mips(include_purecap=True):
-            target_arch = self.config.mips_float_abi.freebsd_target_arch()
-            if self.crosscompile_target.is_cheri_purecap():
-                target_arch += "c" + self.config.mips_cheri_bits_str  # build purecap
```

Comment:
Hm, I guess I need to inherit this special case :( the legacy of CHERI-256 lives on...